Magento_CatalogSearch/css/source/_module.scss

```diff
@@ -29,6 +29,8 @@ $autocomplete-item-amount__color: $color-gray60;
     .label {
         @include lib-icon-font( $_icon-font-content : $icon-search, $_icon-font-size : 22px, $_icon-font-line-height : 28px, $_icon-font-margin : 0 $indent__s 0 0, $_icon-font-color : $minicart-icons-color, $_icon-font-color-hover : $minicart-icons-color-hover, $_icon-font-color-active: $minicart-icons-color-hover, $_icon-font-text-hide : true );
         display: inline-block;
+        float: right;
```

Comment:
`float` + `inline-block` makes no sense at all, b/c every floated element is `block` - check CI tests error.